www/docs/customization/ko.md

```diff
@@ -58,6 +65,18 @@ kos:
   - latest
   - '{{.Tag}}'
 
+  # Creation time given to the image.
```

Comment:
this needs to be seconds since epoch, right? should we make that more explicit here?

Reply:
Correct, I have clarified that in the comment, let me know if that's enough :)